Sources/Kevin/Accounts/BankSelection/KevinBankSelectionView.swift

```diff
@@ -121,10 +123,36 @@ internal class KevinBankSelectionView : KevinView<KevinBankSelectionState> {
         
         bankTableView.translatesAutoresizingMaskIntoConstraints = false
         bankTableView.topAnchor.constraint(equalTo: bankSelectionLabel.bottomAnchor, constant: 16).isActive = true
-        bankTableView.bottomAnchor.constraint(equalTo: continueButton.topAnchor, constant: -20).isActive = true
         bankTableView.widthAnchor.constraint(equalTo: self.widthAnchor).isActive = true
     }
     
+    private func initAgreementLabel() {        
```

Comment:
Name of this function suggest that agreement label is being initialised here, but it seems that values are being set and is layout is being defined. Maybe name it something more appropriate?

Reply:
You have a point, configure probably would make sense better, but then we need to review semantics for the whole project. Let's do that separately, not in this PR.